deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -261,9 +261,6 @@ spec:
                   type: object
                 endpointSecretName:
                   type: string
-              required:
-              - endpointEncryption
-              - endpointSecretName
```

Comment:
just to check but is this change going to affect endpoint encryption CRs?

Reply:
No, those fields are not actually required. We have cr verification in 2.0.0 se we had to correctly specify they are optional. Usage is the same.